src/protocol/http/src/main/java/org/apache/jmeter/protocol/http/sampler/HTTPAbstractImpl.java

```diff
@@ -246,7 +246,6 @@ protected InetAddress getIpSourceAddress() throws UnknownHostException, SocketEx
                 ipClass = Inet6Address.class;
                 break;
             case HOSTNAME:
-            default:
```

Comment:
Is this correct?

Reply:
This is a switch over JMeter's private enum, so it should be pretty safe.